.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,11 @@
+---
+name: Questions
+about: Questions and answers indirectly related to APISIX
```

Comment:
Why use "indirectly"?

Reply:
I was trying to express that these questions are not caused by APISIX, but may be encountered in the course of using APISIX.

Reply:
Maybe we can remove it to make this less confusing? It's hard to distinguish the borderline between directly and indirectly.

Reply:
got

Reply:
update